server/parameters/CoreConfig.java

```diff
@@ -204,6 +209,13 @@ public static class File extends Type {
                     public Path baseDirectory() {
                         return baseDirectory;
                     }
+                    public String filename() {
+                        return filename;
+                    }
+
+                    public String extension() {
+                        return extension;
+                    }
```

Comment:
Filename awareness allows it to be re-used as audit log outputs.
The question is whether this class should be moved to the top level of the config (`CoreConfig.Output`) or remain at `CoreConfig.Log.Output`

Reply:
I dont think we should be reusing the classes across different places (Log/Audit). This is my main discomfort right now and i'm not sure how to resolve it...

Reply:
let's move the output definitions to a `CoreConfig.Common` which will hold structures used by multiple other parts of the definitions?